plugins/json_processor/src/json_processor/component.py

```
@@ -0,0 +1,228 @@
+                ),
+                "_stage_name": getattr(self.config, "stage_name", "unknown"),
+                "_run_id": getattr(self.config, "run_id", "unknown"),
+            }
+
+            for col_name, value in metadata_columns.items():
+                data = data.with_columns(pl.lit(value).alias(col_name))
+
+            logger.debug("Added processing metadata columns")
+            return data
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)